packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -51,7 +51,7 @@ Object {
     "colorBackgroundWeaker": "#F7F9FF",
     "colorBackgroundWeakest": "#F8FAFC",
     "colorBorder": "#64748B",
-    "colorBorderError": "#DC2626",
+    "colorBorderError": "#EF4444",
```

Comment:
I checked in the `dist/variables.js` and it has `#EF4444` in there, but I think it should be `#DC2626` just because it is what we have on Figma 🤷🏻‍♀️

Reply:
@apvale looks like a copy/paste error on my part. I'll get that fixed.